docs/UserGuide.md

```diff
@@ -183,4 +260,26 @@ There is no need to save manually.
 * **Select** : `select INDEX` <br>
   e.g.`select 2`
 
+* **Update** : '`update INDEX [NAME] [d/ENDDATE] [sd/STARTDATE] [r/PERIOD(DAY/WEEK/MONTH)] [t/TAG]...'
```

Comment:
I see. In the future we might wanna get rid of <key>/<value> syntax to come up with a more natural format tho. `add <task> d/<date>` can be `add <task> by <date>`